### PR TITLE
Use JS string templates in src/library_glemu.js. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: src/library_glemu.js [hunk 1]
```diff
@@ -1075,14 +1075,14 @@ var LibraryGLEmulation = {
...
+            return abort_sanity(`Unknown texType: ${ptrToString(texType)}`);
         }
 
         var texCoordExpr = TEX_COORD_VARYING_PREFIX + texUnitID;
         if (TEX_MATRIX_UNIFORM_PREFIX != null) {
-          texCoordExpr = "(" + TEX_MATRIX_UNIFORM_PREFIX + texUnitID + " * " + texCoordExpr + ")";
+          texCoordExpr = `(${TEX_MATRIX_UNIFORM_PREFIX}${texUnitID} * ${texCoordExpr})`;
         }
-        return func + "(" + TEX_UNIT_UNIFORM_PREFIX + texUnitID + ", " + texCoordExpr + ".xy)";
+        return `${func}($[TEX_UNIT_UNIFORM_PREFIX}${texUnitID}, ${texCoordExpr}.xy)`;
```
Comment (brendandahl) [inline src/library_glemu.js:1085]:
```suggestion
        return `${func}(${TEX_UNIT_UNIFORM_PREFIX}${texUnitID}, ${texCoordExpr}.xy)`;
```

`[` looks off

File: src/library_glemu.js [hunk 2]
```diff
@@ -1499,58 +1499,29 @@ var LibraryGLEmulation = {
...
-              ";",
-            ];
-            lines = [line.join("")];
+            lines = [`${outputType} ${outputVar} = ${src0Expr} - ${src1Expr};`]
             break;
           }
           case GL_INTERPOLATE: {
-            var prefix = TEXENVJIT_NAMESPACE_PREFIX + 'env' + texUnitID + "_";
+            var prefix = `${TEXENVJIT_NAMESPACE_PREFIX}env${texUnitID}_`;
             var arg2Var = prefix + "colorSrc2";
```
Comment (brendandahl) [inline src/library_glemu.js:1519]:
```suggestion
            var arg2Var = `${prefix} colorSrc2`;
```

**Review by brendandahl — APPROVED** [review]
r+ with fixes

File: src/library_glemu.js (hunk 2 above)
Comment (sbc100) [reply src/library_glemu.js]: There is no space so I figured this was simple concatenation, but update for consistency with surrounding code.

**Pull request merged**